cmux-tui/bindings/python/tests/test_protocol.py

```
@@ -140,6 +140,54 @@ def test_socket_discovery_precedence_matches_transport_spec(self) -> None:
+            os.environ,
+            {"XDG_RUNTIME_DIR": "/run/user/501", "TMPDIR": ""},
+            clear=True,
+        ):
+            self.assertEqual(
+                default_socket_path(session),
+                f"/run/user/501/cmux-tui-hashed-{os.getuid()}/{digest}.sock",
+            )
+
+        with patch.dict(
+            os.environ,
+            {"XDG_RUNTIME_DIR": "/tmp/" + "x" * 200},
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)